vars/packageBuildingPipelineDAOS.groovy

```diff
@@ -0,0 +1,621 @@
+                                              ls -l $mockroot/result/
+                                              cat $mockroot/result/{root,build}.log
+                                              artdir=$PWD/artifacts/leap42.3
+                                              cp -af _topdir/SRPMS $artdir
+                                              (cd $mockroot/result/ &&
+                                               cp -r . $artdir)
+                                              (if cd $mockroot/root/builddir/build/BUILD/*/; then
+                                                   find . -name configure -printf %h\\\\n | \
+                                                   while read dir; do
+                                                       if [ ! -f $dir/config.log ]; then
+                                                           continue
+                                                       fi
+                                                       tdir="$artdir/autoconf-logs/$dir"
+                                                       mkdir -p $tdir
+                                                       cp -a $dir/config.log $tdir/
+                                                   done
+                                               fi)'''
+                            }
+                            cleanup {
+                                archiveArtifacts artifacts: 'artifacts/leap42.3/**'
+                            }
+                        }
+                    } //stage('Build on Leap 42.3')
+                    stage('Build on Leap 15') {
+                        when {
+                            beforeAgent true
+                            allOf {
+                                expression { distros.contains('leap15')}
+                                expression { return env.QUICKBUILD == '1' }
+                            }
+                        }
+                        agent {
+                            dockerfile {
+                                filename 'packaging/Dockerfile.mockbuild'
+                                label 'docker_runner'
+                                args  '--group-add mock' +
+                                      ' --cap-add=SYS_ADMIN' +
+                                      ' --privileged=true'
+                                additionalBuildArgs '--build-arg UID=$(id -u)' +
+                                                    ' --build-arg JENKINS_URL=' +
+                                                    env.JENKINS_URL
+                            }
+                        }
+                        steps {
+                            sh label: "Build package",
+                               script: '''rm -rf artifacts/leap15/
+                                          mkdir -p artifacts/leap15/
+                                          make CHROOT_NAME="opensuse-leap-15.1-x86_64" ''' +
```

Comment:
Why is this specifically leap 15.1 and not leap 15 to be Leap current?

How do we specify which Leap and which CentOS release to build?

Reply:
[ As discussed on Skype, but just to answer the questions for the record...]

> Why is this specifically leap 15.1 and not leap 15 to be Leap current?

Because unlike CentOS, where [mock](https://github.com/rpm-software-management/mock) understands it my [major version, 7](https://github.com/rpm-software-management/mock/blob/devel/mock-core-configs/etc/mock/epel-7-x86_64.cfg) it knows leap by it's [major and minor](https://github.com/rpm-software-management/mock/blob/devel/mock-core-configs/etc/mock/opensuse-leap-15.1-x86_64.cfg).  Arguably a [bug](https://github.com/rpm-software-management/mock/issues/386).
> 
> How do we specify which Leap and which CentOS release to build?

It's in the `CHROOT_NAME`, if I'm understanding your question.